package.json

```diff
@@ -13,11 +17,15 @@
     "@11ty/eleventy-plugin-syntaxhighlight": "3.1.2",
     "eleventy-plugin-toc": "1.1.0",
     "js-yaml": "4.1.0",
+    "cheerio": "1.0.0-rc.12",
```

Comment:
nits: where does this dep come from?

latest version is [`cheerio@1.1.2`](https://www.npmjs.com/package/cheerio)